app/views/research_sessions/questions/researcher.html.erb

```diff
@@ -1,43 +1,52 @@
-          sheet.
-        </p>
+<%= c('Step') do %>
+  <%= c('Title', text: 'Researcher') %>
+  <%= c('Paragraph') do %>
+    The researcher’s details will go directly on the sheet sent to participants of this session so they can contact you with any queries. The researcher should refer to the safeguarding code of conduct in <%= c('Link', src: 'http://b-hive.barnardos.org.uk/Interact/Pages/Content/Document.aspx?id=5100&SearchId=', text:'BREC Guidance') %> so they are confident they know what to do if a safeguarding concern arises during the session.
+  <% end %>
 
-        <div class="reactive-container">
-          <div class="reactive-container__form">
-            <% previewed_on_page = { 'data-previewed-by' => 'ResearcherPreviews' } %>
-            <%= form.labelled_text_field :researcher_job_title,
-                text_options: { autofocus: true }.merge(previewed_on_page) %>
-            <%= form.labelled_text_field :researcher_name, text_options: previewed_on_page %>
-            <%= form.labelled_text_field :researcher_phone, text_options: previewed_on_page %>
-            <%= form.labelled_text_field :researcher_email, text_options: previewed_on_page %>
-          </div>
-          <div class="reactive-container__preview">
-            <%#
-              react_component(
-                'ResearcherPreviews/index',
-                current_step_params.merge(you_or_your_child: 'you')
-              )
-            %>
-          </div>
-        </div>
+  <%= c('HeadingOne', text: 'Who will the participants meet as part of the session?') %>
+  <%= c('Paragraph') do %>
+    Who is the person who is leading the research or participation session?
+    Please note: participants will be informed that they can get further
+    information from this person, and their contact details will be included
+    on the information sheet.
+  <% end %>
 
-      </fieldset>
+  <%= c('ErrorSummary', errors: @research_session.errors.to_hash) if @research_session.errors.any? %>
 
-      <%= render 'button_bar' %>
+  <%= c('Form', action: wizard_path) do %>
+    <%= c('SinglelineTextControl',
+      id: 'research_session[researcher_job_title]',
+      name: 'researcher_job_title',
+      label: t('research_session_attr_labels.researcher_job_title'),
+      size: 24,
+      errored: @research_session.errors['researcher_job_title'].present?,
+      validation: @research_session.errors['researcher_job_title'][0]
+    ) %>
+    <%= c('SinglelineTextControl',
+      id: 'research_session[researcher_name]',
+      name: 'researcher_name',
+      label: t('research_session_attr_labels.researcher_name'),
+      size: 24,
+      errored: @research_session.errors['researcher_name'].present?,
+      validation: @research_session.errors['researcher_name'][0]
+    ) %>
+    <%= c('SinglelineTextControl',
+      id: 'research_session[researcher_phone]',
+      name: 'researcher_phone',
+      label: t('research_session_attr_labels.researcher_phone'),
+      size: 16,
```

Comment:
This is something we can discuss in more detail during the Rails cleanup that will need to happen but I think these sizes could be abstracted out into something that can provide more meaning.

`SHORT_INPUT`